tools/buildtools/ambertools/tleap.xml

```diff
@@ -0,0 +1,2043 @@
+            <param name="arg_OR" label="OR value" type="text" value="" help=""/>
+            <param name="arg_all" label="all value" type="text" value="" help=""/>
+            <param name="arg_OR" label="OR value" type="text" value="" help=""/>
+            <param name="arg__" label="* value" type="text" value="" help=""/>
+            <param name="arg_showdefault_arg_6" label="showdefault_arg_6 value" type="text" value="" help=""/>
+       </when>
+       <when value="solvateBox">
+            <param name="arg_solute" label="solute value" type="text" value="" help="UNIT"/>
+            <param name="arg_solvent" label="solvent value" type="text" value="" help="UNIT"/>
+            <param name="arg_buffer" label="buffer value" type="text" value="" help="object"/>
+            <param name="arg_solvateBox_arg_3" label="solvateBox_arg_3 value" type="text" value="" help=""/>
+            <param name="arg_iso" label="iso value" type="text" value="" help=""/>
+            <param name="arg_solvateBox_arg_5" label="solvateBox_arg_5 value" type="text" value="" help=""/>
+            <param name="arg_solvateBox_arg_6" label="solvateBox_arg_6 value" type="text" value="" help=""/>
+            <param name="arg_closeness" label="closeness value" type="text" value="" help="NUMBER"/>
+            <param name="arg_solvateBox_arg_8" label="solvateBox_arg_8 value" type="text" value="" help=""/>
+       </when>
+       <when value="solvateCap">
+            <param name="arg_solute" label="solute value" type="text" value="" help="UNIT"/>
+            <param name="arg_solvent" label="solvent value" type="text" value="" help="UNIT"/>
+            <param name="arg_position" label="position value" type="text" value="" help="object"/>
+            <param name="arg_radius" label="radius value" type="text" value="" help="NUMBER"/>
+            <param name="arg_solvateCap_arg_4" label="solvateCap_arg_4 value" type="text" value="" help=""/>
+            <param name="arg_closeness" label="closeness value" type="text" value="" help="NUMBER"/>
+            <param name="arg_solvateCap_arg_6" label="solvateCap_arg_6 value" type="text" value="" help=""/>
+       </when>
+       <when value="solvateDontClip">
+            <param name="arg_solute" label="solute value" type="text" value="" help="UNIT"/>
+            <param name="arg_solvent" label="solvent value" type="text" value="" help="UNIT"/>
+            <param name="arg_buffer" label="buffer value" type="text" value="" help="object"/>
+            <param name="arg_solvateDontClip_arg_3" label="solvateDontClip_arg_3 value" type="text" value="" help=""/>
+            <param name="arg_closeness" label="closeness value" type="text" value="" help="NUMBER"/>
+            <param name="arg_solvateDontClip_arg_5" label="solvateDontClip_arg_5 value" type="text" value="" help=""/>
+       </when>
+       <when value="solvateOct">
+            <param name="arg_solute" label="solute value" type="text" value="" help="UNIT"/>
```

Comment:
Would it make sense to use way more macros here? Those params are used a lot everywhere.

Reply:
corrected